config.ini.example

```diff
@@ -14,6 +14,9 @@ ApiHash = 0123456789abcdef0123456789abcdef
 # Your phone number. You must supply this. Should start with +country code eg +44.
 PhoneNumber = xxxxxxxxxx
 
+# Your 2FA password, if necessary.
+SecondFactorPassword = xxxxxxxxxx
```

Comment:
Not sure if we should call it that or just `Password`. I guess it could be misleading (perhaps it was a password for the database).